### PR TITLE
feat(frontend): do not show news attached files

### PR DESCRIPTION
**Pull request by gbonaspetti**
#4655

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend/src/component/Activity/ActivityList.jsx [hunk 1]
```diff
@@ -130,22 +130,26 @@ const ActivityList = (props) => {
...
-        !activity.newestMessage.fields.workspace) return true
+      !activity.newestMessage.fields.workspace) return true
     const activityWorkspace = props.workspaceList.find(ws => ws.id === activity.newestMessage.fields.workspace.workspace_id)
     if (!activityWorkspace) return true
     return activityWorkspace.publicationEnabled
   }
 
   const isLoggedUserMember = (activity) => props.workspaceList.find(space => space.id === activity.newestMessage.fields.workspace.workspace_id)
 
+  const isAttachedFileOnPublication = (activity) => activity.content
```
Comment (Skylsmoi) [inline frontend/src/component/Activity/ActivityList.jsx:141]: isn't "publication" the french translation for "news"? I think the feature regarding code should be named "activity", like you did in the parameter

File: frontend/src/container/withActivity.jsx [hunk 2]
```diff
@@ -247,11 +247,15 @@ const withActivity = (WrappedComponent, setActivityList, setActivityNextPage, re
       return currentWorkspace.publicationEnabled
     }
 
+    isAttachedFileOnPublication = (activity) => activity.content
```
Comment (Skylsmoi) [inline frontend/src/container/withActivity.jsx:250]: same comment about naming

File: functionnal_tests/cypress/integration/activity_feed/hidden_attached_file_activity_spec.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+const contentType = 'image/png'
+
+describe('Recent activities page', () => {
+  before(function () {
+    cy.resetDB()
+    cy.setupBaseDB()
+    cy.loginAs('administrators')
+    cy.fixture('baseWorkspace').as('workspace').then(workspace => {
+      workspaceId = workspace.workspace_id
+      cy.createPublication(publicationLabel, workspaceId).then(publication => {
```
Comment (Skylsmoi) [inline functionnal_tests/cypress/integration/activity_feed/hidden_attached_file_activity_spec.js:17]: I see "createPublication", I'm then confused by which name it actually is in the code

File: frontend/src/component/Activity/ActivityList.jsx (hunk 1 above)
Comment (Skylsmoi) [reply frontend/src/component/Activity/ActivityList.jsx:141]: actually, the french translation is "Actualité"
but all theses words point to the same feature. What is the official word to use in the code, activity or publication?

Comment (gbonaspetti) [reply frontend/src/component/Activity/ActivityList.jsx:141]: Explained irl, I will update the variable to `isActivityAnAttachedFileOnPublication`

**Review by Skylsmoi — APPROVED** [review]

**Comment by wwalden** [comment]
tested on branch

**Pull request merged**